models/src/main/scala/coop/rchain/models/rholang/sort/ChannelSortMatcher.scala

```diff
@@ -13,5 +13,8 @@ object ChannelSortMatcher {
       case ChanVar(par) =>
         val sortedVar = VarSortMatcher.sortMatch(par)
         ScoredTerm(ChanVar(sortedVar.term), Node(Score.CHAN_VAR, sortedVar.score))
+      case Empty =>
+        ScoredTerm(Empty, Leaf(Score.ABSENT))
```

Comment:
Because there's no way to report errors from the sorter (no `MonadError`) I've decided to score the term as `ABSENT`. This should really never happen.